easybuild/main.py

```diff
@@ -284,6 +284,15 @@ def main(args=None, logfile=None, do_build=None, testing=False, modtool=None):
     elif options.install_github_token:
         install_github_token(options.github_user, silent=build_option('silent'))
 
+    elif options.list_prs:
+        parameters = {
+            'direction': options.list_prs[2],
+            'sort': options.list_prs[1],
+            'state': options.list_prs[0],
+            }
+        print_msg("Listing PRs with parameters %s" % parameters)
+        print list_prs(parameters)
```

Comment:
Just pass down `options.list_prs` to `list_prs`, and unpack there?

`main.py` is big enough as is, let's not make it bigger without good reason :)